### PR TITLE
fix: `index.rst` and migration guide title

### PR DESCRIPTION
**Pull request by Revathyvenugopal162**

### CODE REVIEW

File: doc/source/index.rst [hunk 1]
```diff
@@ -1,3 +1,7 @@
+
+Ansys actions
```
Comment (jorgepiloto) [inline doc/source/index.rst:2]: This is already included from the `README.rst`, right?

Comment (jorgepiloto) [reply doc/source/index.rst:2]: Oh! I see it is no longer present. Thanks for adding this.

Comment (Revathyvenugopal162) [reply doc/source/index.rst:2]: https://github.com/ansys/actions/pull/588 with this PR, the directive starts after the title, so there is no title for the action in the stable docs
image

Comment (jorgepiloto) [inline doc/source/index.rst:2]:
```suggestion
Ansys actions
```

**Review by jorgepiloto — APPROVED** [review]
Thanks! Merging and cherry-picking.

**Pull request merged**